### ISSUE TITLE
Adding similarity measures based on list and count of publications per source_id

### ISSUE THREAD

**Issue by stefano-baruffaldi** [sosia-dev/sosia#19]
This should be easy and does not require additional download of data

**Comment by Michael-E-Rose** [comment]
Duplicate to #2, no?

**Comment by stefano-baruffaldi** [comment]
No, I think this is even more simple than #2. Simply looking at the actual publications of the authors, not at their references, and computing the probability that the journal is the same. Also, not urgent

**Comment by Michael-E-Rose** [comment]
But the other issue is about both cited and citing journals? This seems the other journal is more inclusive, no?

**Comment by Michael-E-Rose** [comment]
It's the number of publications in the same journal. This needs to be implemented in [`inform_matches()`](https://github.com/sosia-dev/sosia/blob/master/sosia/processing/extracting.py#L159). Ideally, we construct it in such a way that is interpreted as "The number of publications in sources the original scientist publishes in up until the comparison year."
